lib/Myriad/RPC/Implementation/Redis.pm

```diff
@@ -110,9 +110,9 @@ async method listen () {
                         await $self->drop($rpc->{stream}, $_->{id});
                     }
                 }
-            }), foreach => [$self->rpc_list->@*], concurrent => 8);
+            }
+        }), foreach => [$self->rpc_list->@*], concurrent => scalar $self->rpc_list->@*);
             await $self->loop->delay_future(after => 0.001);
```

Comment:
Presumably this no longer does anything useful?

Reply:
you mean because this `fmap` will never be finished?
the thing is we need them to run concurrently and without effecting or depending on each others.

Currently in the way they are where while is outside fmap, it will cause them to be closely coupled together

Reply:
oh or I think you meant `await $self->loop->delay_future(after => 0.001);`
yes I totally agree.

which makes wonder on another note why don't we use `$loop->later` instead